contracts/command-batcher/src/msg.rs

```diff
@@ -1,13 +1,32 @@
+use std::collections::HashMap;
+
+use connection_router::types::ChainName;
 use cosmwasm_schema::{cw_serde, QueryResponses};
-use cosmwasm_std::HexBinary;
+use cosmwasm_std::{HexBinary, Uint256, Uint64};
+
+use crate::{encoding::Data, types::ProofID};
 
-use crate::types::{Data, Proof};
+#[cw_serde]
+pub struct InstantiateMsg {
+    pub gateway_address: String,
+    pub multisig_address: String,
+    pub service_registry_address: String,
+    pub destination_chain_id: Uint256,
+    pub signing_threshold: (Uint64, Uint64),
```

Comment:
There is a `Threshold` type in axelar-wasm-std, should use that

Reply:
this is the unvalidated message, the Threshold type is used after validation to store it in the config

Reply:
ah I see. Can you try just adding this to the threshold type:
```
#[serde(try_from="(Uint64, Uint64)")]
pub struct Threshold {
```
https://serde.rs/container-attrs.html#try_from

Reply:
this worked, here is the commit: https://github.com/axelarnetwork/axelar-amplifier/pull/50/commits/1c7491909b20f13e1a08c5651bc9130fa3f03f62. I had to add `into` as well to match both serialization and deserialization otherwise it fails. With this change it will mean that whenever we serialize, the JSON output will be a tuple instead of the Threshold struct, not sure if that's fine for whoever will read the serialized JSON. Thoughts?